### PR TITLE
ci: do not bootstrap twice

### PR DESCRIPTION
**Pull request by emillon**
`make bootstrap` does exactly `make -B _boot/dune.exe`, which is done just before, so we bootstrap twice in the workflow.

### CODE REVIEW

**Comment by emillon** [comment]
I'm not sure what the history is here. This second bootstrap has been added in #6308; the original command has been added in #4327 but with this profile selected, dune is not really rebuilt anyway.

**Comment by rgrinberg** [comment]
I think the purpose of this command is to ensure that the .install file is generated correctly. Could you confirm that the .install file is generated without this command?

**Review by rgrinberg — APPROVED** [review]
LGTM.

File: .github/workflows/workflow.yml [hunk 1]
```diff
@@ -93,10 +93,7 @@ jobs:
 
       # dune doesn't have any additional dependencies so we can build it right
       # away this makes it possible to see build errors as soon as possible
-      - run: opam exec -- make _boot/dune.exe
```
Comment (rgrinberg) [inline .github/workflows/workflow.yml:96]: Good to see another path eliminated from the yaml file. These are always a pain to keep in sync

**Pull request merged**